adapters/mqtt-vertx/readme.md

```diff
@@ -0,0 +1,50 @@
+This module contains a protocol adapter that exposes an MQTT API for Hono's Telemetry API.
+
+# Running the Adapter
+
+The adapter can be run by means of running the corresponding Docker image.
+
+    $ docker run -d --name mqtt-adapter -p1883:1883 eclipsehono/hono-adapter-mqtt-vertx:0.5-SNAPSHOT
```

Comment:
maybe better use `LATEST` instead of `0.5-SNAPSHOT`? Or at least `0.5-M2-SNAPSHOT`?